.github/workflows/main.yml

```diff
@@ -7,8 +7,6 @@ on:
     tags:
     - v*
   pull_request:
-    branches:
-    - dev
```

Comment:
Shouldn't we use `master` here?

Reply:
If we use it here, it means only PRs based on the `master` branches will have the tests run.

I know that you will be suggesting some processual changes that could include a suggestion to stop stacking up PRs. but I would say that, although we can surely try to avoid stacking them up, we might not be able to in some situations, unless we accept that the PRs could get bigger in order to not be stacked up.

So, I would say the rules here should still support running tests for stacked PRs, even if our process at some point changes to try to prohibit this practice.

Reply:
Got it. Ok. We can let it run for all PRs, regardless of the destination branch.